docs/archive/Objective-CInteroperability.rst

```diff
@@ -496,21 +496,21 @@ Overloading
   name, unless using the "selector syntax" for defining methods::
 
     // 1. foo:baz:
-    func foo(Int bar, Int baz);
+    func foo(_ bar: Int, baz: Int)
```

Comment:
I applaud you for finding and fixing these.

@gribozavr What do you think about these changes? It's an archived document with old syntax.